man/systemctl.xml

```diff
@@ -533,11 +532,9 @@
         <term><option>--message=</option></term>
 
         <listitem>
-          <para>When used with <command>halt</command>,
-          <command>poweroff</command>, <command>reboot</command> or
-          <command>kexec</command>, set a short message explaining the reason
-          for the operation. The message will be logged together with the
-          default shutdown message.</para>
+          <para>When used with <command>halt</command>, <command>poweroff</command> or <command>reboot</command>, set a
```

Comment:
Right, it seems that kexec does not set wall. It probably should, but that's a different issue.

Reply:
logind handles the wall message, so it's not a completely separate issue, given this PR is adding setting the wall message to `halt` by redirecting `halt` to logind.

I would agree we shouldn't hold this PR for a decision on `kexec` though.

Also `warn_wall()` is only called from a timer set up for scheduled shutdowns (`/usr/sbin/shutdown`) anyway.  I thought I had a doc fix queued up relating to this, but I can't find anything.  I think technically the message is used outside of the scheduled shutdown method, but only as a log message, not as a wall message.

I'm not sure the final "going down for %s NOW" works either.  That one didn't seem to show up when I was playing with `shutdown -H`.

Reply:
kexec currently isn't supported by "shutdown" or logind, and hence unlike the "halt" case I see no need to add it to logind. I'd rather have fewer of these low-level verbs in place than more...

We might indeed be missing some wall messages in some cases, where we probably should. But I think that should be dealt with in a different issue and PR